bin/subtitle

```diff
@@ -9,13 +9,17 @@ global_opts = Optimist::options do
   opt :access_key_id, "AWS Key", :type => :string, :short => "k"
   opt :secret_access_key, "AWS Secret", :type => :string, :short => "s"
   opt :profile, "AWS Profile", :type => :string, :short => "p"
+  opt :region, "AWS region", :type => :string, :short => "r"
```

Comment:
If we start to get region from options then we need to find a way to pass it all the way upto aws.rb or update the ENV variable `AWS_DEFAULT_REGION`

Reply:
Yes It is done.